src/main/java/com/likeservice/Exception/ServiceExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.likeservice.Exception;
+
+import com.likeservice.Errors.ApiError;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.ServletWebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+public class ServiceExceptionHandler extends ResponseEntityExceptionHandler {
+
+    @ExceptionHandler({LikesNotFoundException.class})
+    ResponseEntity customerNotFoundHandler(Exception exception, ServletWebRequest request){
+        ApiError apiError = new ApiError();
+        apiError.setMessage(exception.getMessage());
+        apiError.setCode("404");
```

Comment:
extract 404 to constants so that it will be helpful incase of writing a test-case

Reply:
Changes Done